packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/Databases.tsx

```diff
@@ -36,39 +35,93 @@ export default function Container() {
   return <DatabaseList {...state} />;
 }
 
+function getEmptyTableText(status: AttemptStatus) {
+  switch (status) {
+    case 'error':
+      return 'Failed to fetch databases.';
+    case '':
+      return 'Searching…';
+    case 'processing':
+      return 'Searching…';
+    case 'success':
+      return 'No databases found.';
+  }
+}
+
 function DatabaseList(props: State) {
```

Comment:
This component seems to fetch the databases of the root cluster after I switch to a leaf cluster. Servers and kubes seem to work fine.